babel-plugin.js

```diff
@@ -14,19 +14,23 @@ module.exports = function ({ types: t }) {
             /\/\*[\s\S]*?\*\/|([^\\:]|^)\/\/.*$/gm, '');
           str = str.replace(/\n/ig, '');
           let newCode = ""
-          if (fun.node.type === 'FunctionExpression') {
+          let type = fun.node.type
+          if (type === 'ArrowFunctionExpression') {
+            str = ('function' + str).replace('=> {', '{')
+            type = 'FunctionDeclaration'
```

Comment:
That's incorrect because arrow functions don't have names. Use 'FunctionExpression' instead.